### PR TITLE
[FPMA-448] - URBNAlert pod 

### PR DESCRIPTION
**Pull request by ryang1428**
Custom alert to be used in FP. You can customize pretty much anything via a URBNAlertStyle object.

You interface with URBNAlertController.

Probably easiest to clone the project and view the code that way. The PR is a bit large.

### CODE REVIEW

**Comment by jgrandelli** [comment]
Update the license

**Comment by jgrandelli** [comment]
Write a readme

**Comment by jgrandelli** [comment]
Clean up the podspec

**Comment by jgrandelli** [comment]
delete the assets folder and the .gitkeep from the classes folder

**Comment by jgrandelli** [comment]
Whats the FBSnapshotTestCase?

**Comment by jgrandelli** [comment]
i don't see any examples for passive alerts

**Comment by ryang1428** [comment]
FBSnapshotTestCase was added when I created the pod. I had to remove it.

**Comment by ryang1428** [comment]
If you run the sample project the bottom 4 blue buttons are all passive alerts

**Comment by jgrandelli** [comment]
what is blah.txt?

**Comment by jgrandelli** [comment]
fix warnings in UIImage+ImageEffects

File: Example/Pods/Local Podspecs/URBNAlert.podspec [hunk 1]
```diff
@@ -26,6 +26,7 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '7.0'
   s.requires_arc = true
+  s.dependency 'URBNConvenience', '~> 0.7'
```
Comment (jgrandelli) [inline Example/Pods/Local Podspecs/URBNAlert.podspec]: Bump this to the new version

File: Example/Pods/Target Support Files/Pods-URBNAlert/Pods-URBNAlert-acknowledgements.markdown [hunk 2]
```diff
@@ -23,4 +23,27 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
+
+## URBNConvenience
+
+Copyright (c) 2014 jgrandelli <jasongrandelli@gmail.com>
```
Comment (jgrandelli) [inline Example/Pods/Target Support Files/Pods-URBNAlert/Pods-URBNAlert-acknowledgements.markdown:29]: Update this

File: Example/URBNAlert/URBNViewController.m [hunk 3]
```diff
@@ -23,82 +26,170 @@ - (void)viewDidLoad {
...
+#pragma mark - Active Alert Touches
 - (IBAction)activeAlertTouch:(id)sender {
-    [self.alertController setAlertStyler:nil];
+    URBNAlertViewController *uac = [[URBNAlertViewController alloc] initWithTitle:@"The Title of my message can be up to 2 lines long. It wraps and centers." message:@"And the message that is a bunch of text. And the message that is a bunch of text. And the message that is a bunch of text."];
     
-    [self.alertController showActiveAlertWithTitle:@"My Alert Title that could be 2 lines but no more than 2" message:@"Message and message and message and going on forever and ever." hasInput:NO buttons:@[@"Yes"] buttonTouchedBlock:^(URBNAlertController *alertController, NSInteger index) {
-        [alertController dismissAlert];
-    }];
+    [uac addAction:[URBNAlertAction buttonWithTitle:@"Done" actionType:URBNAlertActionTypeNormal buttonTouched:^(URBNAlertAction *action) {
+        [uac dismiss];
```
Comment (jgrandelli) [inline Example/URBNAlert/URBNViewController.m]: You need to call dismiss? This should happen automatically no?

File: Pod/Classes/URBNAlertStyle.h [hunk 4]
```diff
@@ -9,25 +9,130 @@
...
+
+/**
+ * Opacity of the alert view's shadow
+ */
+@property (nonatomic, strong) NSNumber *alertViewShadowOpacity;
+
+/**
+ * Radius of the alert view's shadow
+ */
+@property (nonatomic, strong) NSNumber *alertViewShadowRadius;
```
Comment (jgrandelli) [inline Pod/Classes/URBNAlertStyle.h:111]: Why NSNUmber for all these?

Comment (ryang1428) [inline Pod/Classes/URBNAlertStyle.h:111]: so if they are not set I know to use the default

**Comment by ryang1428** [comment]
got a readme up on master

File: Pod/Classes/URBNAlertController.h [hunk 5]
```diff
@@ -11,26 +11,30 @@
 #import "URBNAlertStyle.h"
 
 @class URBNAlertController;
+@class URBNAlertConfig;
+@class URBNAlertViewController;
 
 typedef void(^URBNAlertButtonTouched)(URBNAlertController *alertController, NSInteger index);
-typedef void(^URBNAlertPassiveViewTouched)(URBNAlertController *alertController);
+typedef void(^URBNAlertPassiveAlertDismissed)(URBNAlertViewController *alertViewController, BOOL alertWasTouched);
```
Comment (jgrandelli) [inline Pod/Classes/URBNAlertController.h]: Do you still needs this?

Comment (jgrandelli) [inline Pod/Classes/URBNAlertController.h]: If not also remove it from the tests please.

File: Pod/Classes/URBNAlertStyle.h (hunk 4 above)
Comment (jgrandelli) [inline Pod/Classes/URBNAlertStyle.h:111]: Good idea

**Comment by ryang1428** [comment]
@jgrandelli removed that old block. Also updated the unit tests

**Pull request merged**